<!--REQUEST_ID=69439194-9323-403d-bf70-1cb2193ed583-->
Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.49573&attempt_id=5dcdb9d5-201e-4ea3-bd4a-eb89583936ff)

Attempted: python2Packages.simanneal, python3Packages.simanneal, fastjet, herwig

<details><summary>Partial log (click to expand)</summary><p>

```
f951: Warning: command line option '-Wformat=1' is valid for C/C++/ObjC/ObjC++ but not for Fortran
f951: Warning: command line option '-Wformat-security' is valid for C/C++/ObjC/ObjC++ but not for Fortran
clooptools.cc: In function 'int {anonymous}::start_redirection(std::__cxx11::string)':
clooptools.cc:111:12: warning: ignoring return value of 'FILE* freopen(const char*, const char*, FILE*)', declared with attribute warn_unused_result [-Wunused-result]
     freopen(logfilename.c_str(), "a", stdout);
     ~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
make[1]: Leaving directory '/build/Herwig-7.1.4/Looptools'
make: *** [Makefile:520: all-recursive] Error 1
builder for '/nix/store/cxaxm0vql8jmays1nn1wpwap89986n74-herwig-7.1.4.drv' failed with exit code 2
error: build of '/nix/store/cxaxm0vql8jmays1nn1wpwap89986n74-herwig-7.1.4.drv' failed
```
</p></details>

--- a/pkgs/development/libraries/physics/herwig/default.nix
+++ b/pkgs/development/libraries/physics/herwig/default.nix
@@ -31,5 +31,6 @@ stdenv.mkDerivation rec {
     homepage    = https://herwig.hepforge.org/;
     platforms   = stdenv.lib.platforms.unix;
     maintainers = with stdenv.lib.maintainers; [ veprbl ];
+    broken      = stdenv.isAarch64; # doesn't compile: ignoring return value of 'FILE* freopen...
   };
 }

